.github/workflows/deploy.yml

```diff
@@ -13,17 +13,12 @@ jobs:
       BACKEND_URL: ${{ secrets.BACKEND_URL }}
       VUE_APP_GRAPHQL_HTTP: ${{ secrets.VUE_APP_GRAPHQL_HTTP }}
       VUE_APP_REST_HTTP: ${{ secrets.VUE_APP_REST_HTTP }}
+      MAPCONFIG: ${{secrets.MAPCONFIG}}
     steps:
       - uses: actions/checkout@v2
 
-      - uses: schdck/create-env-json@v2
-        id: create-mapconfig
-        with:
-          file-name: 'mapconfig.json'
-          tilesources: ${{ secrets.MAPCONFIG_TILESOURCES }}
-          cadastral: ${{ secrets.MAPCONFIG_CADASTRAL }}
-          center: ${{ secrets.MAPCONFIG_CENTER }}
-          zoom: ${{ secrets.MAPCONFIG_ZOOM }}
+      - name: create mapconfig
+        run: jq -rc -n env.MAPCONFIG > mapconfig.json
```

Comment:
instead of `jq` command, you may use `echo` command like that.

```bash
echo ${env.MAPCONFIG} > mapconfig.json
```

Reply:
indeed it can! I just made the change with this simplification.